cmd/broker/retry/main.go

```diff
@@ -58,6 +58,9 @@ type envConfig struct {
 
 	// Max to 10m.
 	TimeoutPerEvent time.Duration `envconfig:"TIMEOUT_PER_EVENT"`
+
+	MinRetryBackoff time.Duration `envconfig:"MIN_RETRY_BACKOFF" default:"1s"`
+	MaxRetryBackoff time.Duration `envconfig:"MAX_RETRY_BACKOFF" default:"30s"`
```

Comment:
I was wondering if this should be a bit longer, like a few minutes?

Reply:
How long would you suggest? I don't have a preference. I simply don't want to over blocking the queue because of the limitation we have today.

Reply:
Changed to 1m now.

Reply:
I just think if the backoff reaches 30s, it's very likely the issue is not transient, and some longer backoff is probably better.

But just a guess. We can always tune this later. 1min seems good to me.